joomla/joomla-cms
# Media wrap images in figure

Pull Request for Issue #37380 part1 .

### Summary of Changes
Images are always wrapped in a figure tag.

### Testing Instructions
see #37380 part 1. 

Open an article and add images from media. Add mage class, caption, caption class in all variants.


### Actual result BEFORE applying this Pull Request
see #37380 part 1. 


### Expected result AFTER applying this Pull Request
very image has is wrapped i a figure.
All classes are set, 
Caption is only set when a caption is entered


### Documentation Changes Required
